tests/test_nestor.py

```diff
@@ -0,0 +1,234 @@
+"""Tests for NESTOR algorithm, benchmarked against fortran version.
+
+Created on Mon Apr 12 21:14:09 2021
+
+@author: jonathan
+"""
+
```

Comment:
This is not a real `pytest` test? or does pytest handle having a file like this?